### DISCUSSION TITLE
using local node_module packages

### DISCUSSION THREAD

**Discussion by auchers** [quarto-dev/quarto-cli#4299] [Q&A] [answered]
Hi there — I have been playing around ObservableJS notebooks and was wondering if there was anyway to use local `node_modules` library packages instead of pulling from [jsDelivr](https://quarto.org/docs/interactive/ojs/libraries.html#npm)?

Thanks in advance!

**Comment by cscheid** [comment, marked as answer]
Importing from `node_modules` won't really work because although those are also Javascript, nodejs libraries are not compatible with the browser environment. However, if you have local javascript libraries in your project which work in a browser and you want to reuse them, you _can_ import from them directly. See https://quarto.org/docs/interactive/ojs/code-reuse.html. (Do note that they need to be ES modules.)


````
```{ojs}
